nixpkgs_review/utils.py

```diff
@@ -9,7 +10,7 @@
 ROOT = Path(os.path.dirname(os.path.realpath(__file__)))
 
 
-def color_text(code: int, file: IO[Any] = sys.stdout) -> Callable[[str], None]:
```

Comment:
If you are wondering why I changed this:

I needed access to `sys.stdout` for the test cases. For some reason `pytest` cannot access the standard output correctly. (See https://github.com/pytest-dev/pytest/issues/5997 or https://github.com/pytest-dev/pytest/issues/6676) .
The comment https://github.com/pytest-dev/pytest/issues/6676#issuecomment-582393940 mentioned using `None` as output. This still outputs everything correctly to the terminal, but allows the tests to access stdout as well.

As per [the docs](https://docs.python.org/3/library/functions.html#print) this defaults to its original behavior anyway:
>The file argument must be an object with a write(string) method; if it is not present or None, [sys.stdout](https://docs.python.org/3/library/sys.html#sys.stdout) will be used.